### DISCUSSION TITLE
Using with Cloudflare Workers and Hono?

### DISCUSSION THREAD

**Discussion by osseonews** [refinedev/refine#5810] [Q&A] [answered]
I'm wondering if anyone has had the ability to integrate Refine with Cloudflare Workers and Hono? Cloudflare Workers is Cloudflare's serverless environment that runs on the edge. Hono is a framework to help with routing and other features of Workers. you can now use client side code in Hono (https://hono.dev/guides/jsx-dom) and with a react rendere middleware, https://github.com/honojs/middleware/tree/main/packages/react-renderer

**Comment by BatuhanW** [comment, marked as answer]
Hey @osseonews refine doesn't have anything special on top of react-rendering.
